New features

KatieFrogs opened the pull request:
- [x] Added updating the thread when u click on the [Update] button at the bottom of the thread, it is not rly obvious that it is updating, but at least now u do not have to F5 all the time
- [x] Added support for quoting of interface posts, it already sort of worked before, but now it will properly quote in all 6 fields and even works with 4chan-X, it picks the one which u clicked last
- [x] Fixed an issue when opening native extension's QR for the second time would disable green posting

Commit:
Add updating and quoting

Commit:
Fix native extension QR

doctorwth commented:
beautiful. in firefox with the native QR there are still some posting issues (the extension breaks draggability, captcha, and 4chan's inline extension overrides the post's submit event). I have a hacky workaround in the native-qr-fixes branch.

I'll be busy for the next few hours but when I get home tonight I will merge everything together, do all the tests i want to do and then make a thread officially announcing v2.00

Commit:
Add ajax posting, fix classic form, fix native QR

KatieFrogs commented:
- [x] Added ajax posting, now u do not have to leave the thread at all bcus u post and it just werks (btw I hate native QR now bcus it gave me the most trouble like having to make a button out of a `<span>` element otherwise it would submit the post regularly...)
- [x] Fixed (for real this time) native QR (btw I hate native QR even more now)
- [x] Fixed classic post form, turns out it never actually worked before now

Commit:
Updating the thread itself is probably not important

doctorwth commented:
I get the hate for native QR. I spent a good deal of time fugging with it.

thank you bby
